.github/workflows/packages_tests.yaml

```diff
@@ -50,6 +51,10 @@ jobs:
                 run: composer require rector/rector-src dev-main#${{github.event.pull_request.head.sha}} --no-update
                 if: ${{ github.event_name == 'pull_request' }}
 
+            -
+                run: composer require rector/rector-downgrade-php:dev-main --no-update
+                if: github.repository != 'rectorphp/rector-downgrade-php'
```

Comment:
this is temporary as require rector/rector-src in previous step seems using --no-update